aspnetcore/tutorials/first-mvc-app/controller-methods-views.md

```diff
@@ -24,18 +24,24 @@ Open the *Models/Movie.cs* file and add the highlighted lines shown below:
 
 * Right click on a red squiggly line **> Quick Actions**.
 
-    ![image](controller-methods-views/_static/qa.png)
+  ![image](controller-methods-views/_static/qa.png)
+
+  > [!Note]
```

Comment:
DocFX Notes are too obtrusive to use this often.  I think we should replace them with the following MD
**Visual Studio 2017 RC 2 note:** This step isn't necessary. Unneeded using statements were automatically removed.

cc/ @danroth27  @tdykstra